semgrep-core/parsing/Parse_java_tree_sitter.ml

```diff
@@ -50,13 +51,11 @@ module G = AST_generic
 
 [@@@warning "-32"]
 
-type env = unit
+type env = H.env
 
-let token (env : env) (_tok : Tree_sitter_run.Token.t) =
-  failwith "not implemented"
+let token (env : env) (_tok : Tree_sitter_run.Token.t) = H.token
```

Comment:
should this be in a Java parsing PR instead?

Reply:
yes it should ...